sdk/src/pubkey.rs

```diff
@@ -54,11 +66,31 @@ impl fmt::Display for Pubkey {
     }
 }
 
+pub fn write_pubkey(outfile: String, pubkey: Pubkey) -> Result<(), Box<error::Error>> {
```

Comment:
Why a `String` here and not a `&str` like in `read_pubkey()`?

Reply:
Clippy suggested it at one point, but I rearranged things since then. Will fix.